app.test.js

```diff
@@ -43,13 +45,40 @@ describe('Server', () => {
         expectedProject.updated_at = expectedProject.updated_at.toJSON()
 
       const response = await request(app).get(`/api/v1/projects/${id}`)
-      const result = response.body[0]
+      const projectName = response.body[0].project_title
+      expect(projectName).toEqual(expectedProject.project_title)
+    });
 
-      expect(result).toEqual(expectedProject)
-    })
-  })
+    it('should not return a project if there is no match with given id', async () => {
+      const projectId = 0
+      const mockResponse = `No project found with id of ${projectId}`
+      const response = await request(app).get(`/api/v1/projects/${projectId}`)
+      const result = response.body.error
+      expect(result).toEqual(mockResponse)
+    });
+  });
+
+  describe('DELETE /api/v1/projects/:id', () => {
```

Comment:
I know deleting all palettes associated with a project was tricky but thanks for pushing through